src/components/App/App.jsx

```diff
@@ -186,6 +187,7 @@ const App = (props) => {
             <Route exact path="/scenarios/:info?" component={Scenarios} />
             <Route exact path="/predictions" component={Predictions} />
             <Route exact path="/api-keys" component={Api} />
+            <Route exact path="/subscribe" component={Subscription} />
```

Comment:
should we hide this from the nav until we test it first?

Reply:
done